config/api.yml

```diff
@@ -1124,10 +1128,6 @@
       :get:
       - :name: show
       - :identifier: miq_cloud_networks_view
-    :load_balancers_subcollection_actions:
-      :get:
-      - :name: show
-        :identifier: load_balancer_show
```

Comment:
@abellotti yep, I added the refactoring as a separate commit last in https://github.com/ManageIQ/manageiq/pull/13866/commits/91535c514c7fc812db0ef51441db4d5691e28319